tests/test_util.py

```diff
@@ -39,22 +39,18 @@
 )
 from pydicom.util.hexutil import hex2bytes, bytes2hex
 from pydicom.util.leanread import dicomfile
+from ._write_stds import impl_LE_deflen_std_hex
 
 have_numpy = True
 try:
     import numpy
 except ImportError:
     have_numpy = False
 
-test_dir = os.path.dirname(__file__)
-raw_hex_module = os.path.join(test_dir, "_write_stds.py")
-raw_hex_code = open(raw_hex_module, "rb").read()
-
 
 # For Python >=3.11, this can be imported from contextlib
 @contextmanager
 def chdir(new_dir):
-    import os
 
     old_dir = os.getcwd()
```

Comment:
I don't see where `os` is imported now... obviously missing something

Reply:
I see it in line 6 of `test_util.py`.